src/client/bridge/gateway/mod.rs

```diff
@@ -1,4 +1,4 @@
-//! The client gateway bridge is support essential for the [`client`] module.
+//! The client gateway bridge is support essential for the [`client`](../../index.html) module.
```

Comment:
I don't like inline anchors since they're harder to read, and in this case brings the character count above 80. Can we move this back to a named anchor?

Reply:
Also here it is done to differentiate between `client` and `Client`